CMakeLists.txt

```diff
@@ -101,6 +101,7 @@ set(CIVETWEB_ENABLE_SSL_DYNAMIC_LOADING OFF CACHE BOOL "Disable dynamic SSL libr
 set(CIVETWEB_ENABLE_CXX ON CACHE BOOL "Enable civet C++ library")
 add_subdirectory(thirdparty/yaml-cpp-yaml-cpp-0.5.3)
 add_subdirectory(thirdparty/civetweb-1.9.1 EXCLUDE_FROM_ALL)
+add_subdirectory(thirdparty/libarchive-3.3.2)
```

Comment:
Both this and MINIFICPP-72 use libarchive. Not opposed to this being moved to an extension, but it needs to be coordinated between the two.

Reply:
Great! We'll work on getting the other PR in an extension and this can be rebased.

Reply:
I'm not sure how to do that, are there any examples?

Reply:
@calebj I think @achristianson point is valid that it shouldn't be done here. I will submit a PR to the other PR and then when that is merged ( should be the next day ), you will get that for free via the rebase. So feel free to ignore any comment about "extensions" since that is on me. thanks!